docs/data/material/guides/shadow-dom/shadow-dom.md

```diff
@@ -14,7 +14,7 @@ The following code snippet shows how to apply styles inside of the shadow DOM:
 ```tsx
 const container = document.querySelector('#root');
 const shadowContainer = container.attachShadow({ mode: 'open' });
-const emotionRoot = document.createElement('style');
+const emotionRoot = document.createElement('div');
```

Comment:
`style` is invisible by default. I agree that we can go with a `div`, but let's add a hidden attribute to it:

```suggestion
const emotionRoot = document.createElement('div');
emotionRoot.hidden = true;
```

Even though an empty div shouldn't render, developers may have styles that add some margins or otherwise affect the layout. Having it hidden by default should prevent such surprises.